Sources/PayPal/Buttons/PayPalButton.swift

```diff
@@ -0,0 +1,20 @@
+import UIKit
+
+/// Configuration for PayPal button
+public class PayPalButton: PaymentButton {
+
+    // MARK: - Init
+
+    public init() {
+        super.init(frame: .zero)
+
+        translatesAutoresizingMaskIntoConstraints = false
+        backgroundColor = getButtonColor(for: .payPal)
+        setImage(getButtonLogo(for: .payPal), for: .normal)
```

Comment:
`init()` functions for `PayPalButton` and `PayPalCreditButton` look kinda similar now, what do you think of adding an init for `PaymentButton`:
```
func init(type: PayPalButtonType) {
        super.init(frame: .zero)
        translatesAutoresizingMaskIntoConstraints = false
        backgroundColor = getButtonColor(for: type)
        setImage(getButtonLogo(for: type), for: .normal)
}
```

Then call this function in PayPal and Credit button classes?

Reply:
Hmmm, I think it feels a little confusing (at least to me) to call a function init and call it from the sub-classes? Maybe something like `createButton(with type: PayPalButtonType)` would make more sense?

Reply:
hmm imo it seems normal to do so - in line 9 above we also call superclass (UIButton) 's `init` insider subclass (PayPalButton) 's `init` :smile: 
but yea I think either works fine, as long as it reduce code duplication!!!!

Reply:
It seems like calling the func `init` is not best practice as we get the following warning when doing so `Keyword 'init' cannot be used as an identifier here`. Additionally, we wouldn't be able to set `super.init(frame: .zero)` in this function as it cannot be called outside of an initializer. I can certainly make a function to wrap the other 3 aspects, but that seems a little silly.

I think it may make more sense to either make a custom `init` on `PaymentButtons` then call `super.init(type: PaymentButtonType)` on the subclasses. This will make our subclasses really tiny for now until we add more functionality/customization, but maybe that's fine? Thoguhts?

Reply:
Went with the second option and extracted the color and logo into an enum instead of a func to allow more flexibility as we add additional customization options. 😄